example/src/MenuItems.constants.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { Hyperlink } from '@edx/paragon';
+
+export const MENU_ITEMS = [
```

Comment:
I'm thinking it would be useful to include these constants as a part of the library, so we are not copying/pasting this code across frontend-app repos. The file could be called `MarketingSiteMenuItems.constants.js` (this is probably not the right name) or something that identifies that these are the menu items that should be used for a header in the given context.